filters/private/expr/MathParser.cpp

```diff
@@ -167,9 +167,29 @@ bool MathParser::primary(Expression& expr)
 
 bool MathParser::function(Expression& expr)
 {
+    using namespace std;
```

Comment:
This isn't necessary on OSX. Was there a problem on some other platform? Curious.

Reply:
dropped.